major: updates for working backend and frontend

axelra82 opened the pull request:
Highlights:
- Missing fields are added. Signup for CUSTOMER and SUPPLIER now possible.
- Showing list of suppliers on frontpage
- Added missing public hairdresser/:id route

Live app preview @ [netlify](https://deploy-preview-21--ecstatic-poitras-87646f.netlify.app)

## UPDATE 1

It is now possible to rate a hairdresser. Value is saved in ratings table. **NOTE** Updating rating currently uses `POST` which creates new rating entry. This should be put for update (which is available in current main branch (untested). See comment in rating component).

**Process:**
- If client is not logged in they are encouraged to create account or login
- If logged in and no rating: able to set rating
- If logged in and rating: able to edit rating, can't change before clicking edit button

Rating heading changes accordingly.

axelra82 committed:
major: updates for working backend and frontend

Highlights:
- Missing fields are added. Signup for CUSTOMER and SUPPLIER now possible.
- Showing list of suppliers on frontpage
- Added missing public hairdresser/:id route

Fijo-Lurk reviewed (approved):
There is conflicts in backend

axelra82 committed:
major: ratings

Allows for setting and editing ratings on hairdressers.
Editing should use put to update, not post.

axelra82 commented:
> There is conflicts in backend

Yes, they idea is that you can all look at the code in both frontend and backend and extract what you need for your current setup to make it work. Since I built this on an older main you will all have to take note of what lines have changed and what the diff is 👍

Fijo-Lurk commented:
Axel on zoom with Filip. Closing. Use new branch from main.
